src/components/stepThree/CrowdsaleBlock.spec.js

```diff
@@ -0,0 +1,148 @@
+import React from 'react'
+import TierStore from '../../stores/TierStore'
+import { CrowdsaleBlock } from './CrowdsaleBlock'
+import MockDate from 'mockdate'
+import moment from 'moment'
+import { VALIDATION_TYPES } from '../../utils/constants'
+import { Provider } from 'mobx-react'
+import Adapter from 'enzyme-adapter-react-15'
+import toJson from 'enzyme-to-json'
+import { configure, mount } from 'enzyme'
+import { defaultTier, defaultTierValidations } from '../../utils/constants'
+
+configure({ adapter: new Adapter() })
+
+const currentTime = '2018-03-05T11:00:00'
+const { INVALID } = VALIDATION_TYPES
+
+MockDate.set(currentTime)
```

Comment:
I'm not sure how jest tests are run. Should this be in a `beforeAll`? Should the `afterAll` function call a restore?

(If each jest spec is run independently, this may not be necessary, but I'm not really sure about that.)